crates/typst-library/src/math/op.rs

```diff
@@ -21,7 +21,7 @@ use super::*;
 pub struct OpElem {
     /// The operator's text.
     #[required]
-    pub text: EcoString,
+    pub text: Content,
```

Comment:
Is the name `text` still appropriate or should it be `body` then? It's maybe a bit unnecessary to break it and typically it will be `text`...

Reply:
I had the exact same thought. In the end, I decided to stick with `text` out of backwards compatibility, but I'm still not 100% sure what the best solution would be.

Reply:
I think we can leave it as-is.